cilium/cmd/status.go

```diff
@@ -107,7 +107,7 @@ func statusDaemon() {
 			healthPkg.GetAndFormatHealthStatus(w, true, allHealth, healthLines)
 			healthPkg.GetAndFormatModulesHealth(w, client.Daemon, allHealth)
 		} else {
-			fmt.Fprint(w, "Cluster health:\t\tProbe disabled\n")
+			fmt.Fprint(w, "Cluster Health:\t\tProbe disabled\n")
```

Comment:
While I appreciate the intent of these changes, I don't particularly think they should be part of this PR and merged late into the release cycle. My short term concern is that we might have scripts somewhere relying on this. Longer term, without a style guide to point to, these changes are arbitrary and "subjective". Someone later down the line might come along and propose the effective inverse of these changes, :shrug:.

My proposal is to drop this commit for now.

Happy to discuss and pull in the right folks if we think a style guide for our CLI outputs to be more consistent / following a particular pattern is something we should pursue.

Reply:
Makes sense. I'll axe it. We have a bit of a `Frankenstein` theme going at the moment regarding the output with a strong majority to capitalize, hence this effort to normalize the output for consistency. 
That said, If scripts are scraping human readable console format while we have options to use structured data via yaml or json, we have other issues.